core/src/main/scala/scala/collection/parallel/mutable/ParArray.scala

```diff
@@ -472,9 +472,9 @@ self =>
           pac =>
             // with unr. combiner:
             val targetarr: Array[Any] = pac.buff.lastPtr.array.asInstanceOf[Array[Any]]
-          Array.copy(arr, i, targetarr, 0, until - i)
-          pac.buff.size = pac.buff.size + until - i
-          pac.buff.lastPtr.size = until - i
+            Array.copy(arr, i, targetarr, 0, until - i)
+            pac.buff.size = pac.buff.size + until - i
+            pac.buff.lastPtr.size = until - i
```

Comment:
IMO the codebase occasionally places `foo =>` contrary to my expectation, so here I left these lines and instead:
```
-        cb.ifIs[UnrolledParArrayCombiner[T]] {
-          pac =>
-            // with unr. combiner:
-            val targetarr: Array[Any] = pac.buff.lastPtr.array.asInstanceOf[Array[Any]]
+        cb.ifIs[UnrolledParArrayCombiner[T]] { pac =>
+          // with unr. combiner:
+          val targetarr: Array[Any] = pac.buff.lastPtr.array.asInstanceOf[Array[Any]]
           Array.copy(arr, i, targetarr, 0, until - i)
           pac.buff.size = pac.buff.size + until - i
           pac.buff.lastPtr.size = until - i
```

Reply:
better. done